app/controllers/EventReportController.scala

```diff
@@ -46,13 +46,28 @@ class EventReportController @Inject()(
 
   private val submitEventDeclarationReportSchemaPath = "/resources.schemas/api-1828-submit-event-declaration-report-request-schema-v1.0.0.json"
 
-  def saveEvent: Action[AnyContent] = Action.async {
+  def saveUserAnswersToCache: Action[AnyContent] = Action.async {
     implicit request =>
       withPstrEventTypeAndBody { (pstr, eventType, userAnswersJson) =>
         logger.debug(message = s"[Save Event: Incoming-Payload]$userAnswersJson")
         EventType.getEventType(eventType) match {
           case Some(et) =>
-            eventReportService.saveEvent(pstr, et, userAnswersJson).map(_ => Created)
+            eventReportService.saveEventToMongo(pstr, et, userAnswersJson).map(_ => Ok)
```

Comment:
Is this saving event or userAnswers ? 
And do we need to say toMongo explicitly ?

Reply:
Renamed method to make clear